### DISCUSSION TITLE
Blank Screen at launch Solution

### DISCUSSION THREAD

**Discussion by Vinayak-kadu** [FreeTubeApp/FreeTube#5369] [General]
this is the second time encountered this problem, FreeTube suddenly opens with blank screen, cant interact with app.
upon closing, it keeps running error in background, if you reopen it throws Javascript object destroyed error.
previously the solution was simple to update to latest beta, but this time it didn't worked. though updated to 0.21 4551.

this is the solution i found on myself, on my system.
i think FT cant read important files at startup, mainly are History.db, Setting.db, profiles.db.

you can open these with notepad++ and see if multiple NULL values added at somewhere usually at the end. 
in my case it was profiles.db somehow it was not saved correctly and had stream of NULL added to end. 
i simply selected all these chars, deleted them, saved. and relaunch FT. 
Success.! no data loss, all history till last time you watched. 

i suggest all to take back up of these files regularly as a backup and specially before the edit. best luck.

**Comment by philmcrakin** [comment]
Thank you so much for posting this, I was so frustrated trying to get this working again. In linux the file was locate at 
**~/.config/FreeTube/profiles.db**

Some corrupt/null characters were appended to the end of the file as you described above, removing them from the file, saving and relaunching freetube solved my problem without data loss.

Thanks again for sharing this info, you've been a life saver.

**Reply by demidovia** [reply]
Can you please help me to locate where this file is located in Ubuntu?

**Comment by M1CR0H4CK3R** [comment]
Oh, shoot. This fixed the same issue for me as well. Thanks a million! My power went out while loading FreeTube, and this issue happened. I was about to open a discussion about it when I missed the checkbox that says `I have done a search for similar discussions`, so I double-checked and found this. Thanks again!
